clients/python/tests/test_e2e.py

```diff
@@ -92,19 +94,27 @@ def server_url() -> Generator[str]:
 
 
 def test_full_cycle(server_url: str) -> None:
-    client = ClientBuilder(server_url, "test-usecase").for_organization(12345)
+    client = Client(server_url)
+    test_usecase = Usecase(
+        "test-usecase",
+        compression="zstd",
```

Comment:
Since this is the default, let's remove it from this test?
We should add one test with `compression="none"`